src/app/components/skills/skills.component.html

```diff
@@ -1,19 +1,20 @@
-<ng-container *ngFor = 'let skill of skills; let i = index'>
-    <button mat-stroked-button color='basic' *ngIf='skill.isActive'
-        (click) = 'changeActive(i)'>
-        {{skill.skill}}
-    </button>
-    <button mat-raised-button color='primary' *ngIf='!skill.isActive'
-        (click) = 'changeActive(i)'>
-        {{skill.skill}}
-    </button>
-</ng-container>
+<div class="wrapper">
+    <h4>Skills</h4>
+    <ng-container *ngFor='let skill of skills; let i = index'>
+        <button mat-stroked-button color='basic' *ngIf='!skill.isActive' (click)='skillSearch(i)'>
```

Comment:
difference is just on attribute color?

just use smth like this

```
[attr.checked]="value ? '' : null"
[attr.checked]="value ? 'checked' : null"
```

Reply:
difference is on attribute mat-stroked-button too.

Reply:
ok